src/dispatch/auth/models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import (
+    Column,
+    String)
+from dispatch.database import Base
+from dispatch.models import TimeStampMixin, DispatchBase
+from pydantic import validator
+
+
+class DispatchUser(Base, TimeStampMixin):
+    email = Column(String, primary_key=True)
+
+
+class UserLoginForm(DispatchBase):
+    email: str
```

Comment:
Where do you authenticate the user? I don't see where you are storing a password (that you would then exchange for a JWT token).

Reply:
kevgliss, I omitted storing password, I will make the modifications necessary.